xclim/indicators/atmos/_temperature.py

```diff
@@ -72,6 +72,10 @@
     "huglin_index",
     "biologically_effective_degree_days",
     "latitude_temperature_index",
+    "cold_and_dry_days",
```

Comment:
I put these 4 functions in  _temperature.py but as they are related to both temperature and precipitation you may prefer if I move them in another module ?

Reply:
Up until now, we put the tas+pr in `_precip.py`. There is no good reason for this, so you can leave them here if you want.

Though, for explicitness, I suggest creating a `PrecipTemp` class:

```python
class PrecipTemp(Daily):
    """Indicators involving temperature and precipitation."""
    context = "hydro"
```

Where the `context` attribute is a (potentially unneeded) flag for the unit handling. Note that this as virtually no real role, other than making the code coherent.

Reply:
I will leave them here then. I prefer to let you choose and implement the structural changes which affect the existing code base.

Reply:
I think there was a misunderstanding, I am not suggesting a structural change, only that we add that `PrecipTemp` class and use it for the 4 indicators here. The indicators subclass are supposed to reflect what variable types are involved, and `Temp ` is thus incoherent for indicators taking both `tas` and `pr` as input.

Reply:
Ok, I will create PrecipTemp class and move the 4 indices of the Pr in it.
I though you wanted to move all the existing tas+pr indices in `PrecipTemp`, but because I don't know them I was afraid to mess up.